api/lds.proto

```diff
@@ -66,13 +66,27 @@ message FilterChainMatch {
 
   // If non-empty, an IP address and prefix length to match addresses when the
   // listener is bound to 0.0.0.0/::.
```

Comment:
Here's my current understanding. When using `use_original_dst`, the `findListenerByAddress()` method is used to determine the `Listener` based on the original destination address. In LDS, I assume we will follow a similar path. This means we get to the `FilterChainMatch` after either (1) a 0.0.0.0/:: bind or (2) an exact match on the bound address and original destination address. (1) is covered above, with (2) we wouldn't ever specify a prefix/suffix explicitly, since there's only 1 valid IP for the listener.

I might be missing something, so please let me know if that makes sense.